src/bitmessagekivy/baseclass/login.py

```diff
@@ -0,0 +1,115 @@
+# pylint: disable=no-member, too-many-arguments, too-few-public-methods
+# pylint: disable=no-name-in-module, unused-argument, arguments-differ
+
+"""
+Login screen appears when the App is first time starts and when new Address is generated.
+"""
+
+
+from kivy.clock import Clock
+from kivy.properties import StringProperty, BooleanProperty
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.screenmanager import Screen
+from kivy.app import App
+
+import queues
+
+from kivymd.uix.behaviors.elevation import RectangularElevationBehavior
+
+from bitmessagekivy.baseclass.common import toast
+from bmconfigparser import BMConfigParser
+
+
+class Login(Screen):
+    """Login Screeen class for kivy Ui"""
+    log_text1 = (
+        'You may generate addresses by using either random numbers'
+        ' or by using a passphrase If you use a passphrase, the address'
+        ' is called a deterministic; address The Random Number option is'
+        ' selected by default but deterministic addresses have several pros'
+        ' and cons:')
+    log_text2 = ('If talk about pros You can recreate your addresses on any computer'
+                 ' from memory, You need-not worry about backing up your keys.dat file'
+                 ' as long as you can remember your passphrase and aside talk about cons'
+                 ' You must remember (or write down) your  You must remember the address'
+                 ' version number and the stream number along with your passphrase If you'
+                 ' choose a weak passphrase and someone on the Internet can brute-force it,'
+                 ' they can read your messages and send messages as you')
+
+
+class Random(Screen):
+    """Random Screen class for Ui"""
+
+    is_active = BooleanProperty(False)
+    checked = StringProperty("")
+
+    def generateaddress(self):
```

Comment:
a lot of this is common for all frontends so it should be made into a common method/class